sdk/cosmosdb/cosmos/src/index.ts

```diff
@@ -48,7 +48,15 @@ export {
 
 export { UniqueKeyPolicy, UniqueKey } from "./client/Container/UniqueKeyPolicy";
 export { ContainerRequest } from "./client/Container/ContainerRequest";
-export { Constants, OperationType, ResourceType, HTTPMethod } from "./common";
+export {
+  Constants,
+  OperationType,
+  ResourceType,
+  HTTPMethod,
+  CosmosKeyType,
+  CosmosContainerChildResourceKind,
+  SasTokenPermissionKind
+} from "./common";
```

Comment:
```suggestion
export {
  Constants,
  OperationType,
  ResourceType,
  HTTPMethod
} from "./common";
```
It think we can remove these since they aren't needed by any consumer yet